<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1393</width>
    <height>845</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1221</width>
      <height>721</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QWidget" name="Cam_3" native="true"/>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="Cam_2" native="true"/>
     </item>
     <item row="1" column="1">
      <widget class="QWidget" name="Cam_4" native="true"/>
     </item>
     <item row="0" column="0">
      <widget class="QWidget" name="Cam_1" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="thresholdButton">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>2</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Low Threshold</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>132</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Monitoring</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listEvent">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>60</y>
      <width>171</width>
      <height>751</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipCam">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera IP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addCam">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to stream</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1393</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout_us">
    <property name="title">
     <string>About us</string>
    </property>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout_us"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
